web/modules/custom/dpl_article/dpl_article.info.yml

```
@@ -0,0 +1,5 @@
+name: "dpl_article"
```

```suggestion
name: "Article"
```

The name of the module is what will be shown to administrators in the module list. Using a human readable name is always nice.